src/abcdmicro/masks.py

```diff
@@ -140,6 +140,24 @@ def extract_gen_b0_args(
     return args
 
 
+def run_hd_bet(
+    hd_bet_input: list[str], hd_bet_output: list[str], overwrite: bool
+) -> None:
+    """
+    Run HD-BET on the given input files. The expensive HD-BET import and run call are isolated in this function.
+
+    :param hd_bet_input: passed to the `mri_fnames` parameter of HD_BET.run.run_hd_bet
+    :param hd_bet_output: passed to the `output_fnames` parameter of HD_BET.run.run_hd_bet
+    :param overwrite: passed to the `overwrite` parameter of HD_BET.run.run_hd_bet
```

Comment:
Do we want to paste in some content from HD_BET?

Reply:
I think we should leave it like this since this function isn't meant to be used as anything except as an HD_BET caller. The intention for this function is not to do anything  in particular related to brain masking, but rather for it to do whatever HD_BET does. If a user of this package wants to call HD_BET then they should do it using HD_BET, not using this function from our API. This function is just here to isolate the expensive import and allow us to skip it in unit testing.

Reply:
HD_BET doesn't really provide a functional interface. If this is meant to be a functional wrapper, then I'd document it as such. I guess this is just an implementation detail of our tool, but then I'd mark it as such with `_private`.

We could also consider creating a fork of HD-BET, both for version pinning (so our project still works even if upstream is removed) and for the opportunity to add a real functional api. I doubt the effort of an api changes is worth it, probably better to call wrappers like this one an implementation detail.

Reply:
Good points, I might just add a note to the docstring and prepend a `_`. Prepending `_` to the name is understood to be the same as marking `_private`, right?

Reply:
That's what I meant, yes, `_run_hd_bet`.